### PR TITLE
refactor!(support): move opencv related functionality into its own package

### PR DESCRIPTION
**Pull request by jlipps**
here's the follow up to https://github.com/appium/appium/pull/16339, which could just be merged into that PR, which takes the opencv stuff out of support and makes it its own package.

### CODE REVIEW

File: packages/opencv/lib/index.js [hunk 1]
```diff
@@ -0,0 +1,629 @@
...
+      multiple: results
+    };
+  } finally {
+    try {
+      fullImg.delete(); partialImg.delete(); matched.delete();
+    } catch (ign) {}
+  }
+}
+
+function jimpImgFromCvMat (mat) {
```
Comment (mykola-mokhnach) [inline packages/opencv/lib/index.js:581]: it would be nice to add docstrings to these new functions

**Review by mykola-mokhnach — APPROVED** [review]
👍

File: packages/support/lib/image-util.js [hunk 2]
```diff
@@ -3,86 +3,10 @@ import Jimp from 'jimp';
 import { Buffer } from 'buffer';
 import { PNG } from 'pngjs';
 import B from 'bluebird';
-import { hasValue } from './util';
```
Comment (mykola-mokhnach) [inline packages/support/lib/image-util.js:6]: shall we also update images plugin?

File: packages/opencv/lib/index.js (hunk 1 above)
Comment (jlipps) [reply packages/opencv/lib/index.js:581]: ok, added.

**Comment by KazuCocoa** [comment]
👍

**Pull request merged**